supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiJobCachingActorHelper.scala

```diff
@@ -20,7 +20,11 @@ trait PipelinesApiJobCachingActorHelper extends StandardCachingActorHelper {
 
   lazy val pipelinesApiCallPaths: PipelinesApiJobPaths = jobPaths.asInstanceOf[PipelinesApiJobPaths]
 
-  lazy val runtimeAttributes = PipelinesApiRuntimeAttributes(validatedRuntimeAttributes, pipelinesConfiguration.runtimeConfig)
+  lazy val runtimeAttributes = PipelinesApiRuntimeAttributes(
+    validatedRuntimeAttributes,
+    pipelinesConfiguration.runtimeConfig,
+    googleLegacyMachineSelection(jobDescriptor.workflowDescriptor)
```

Comment:
Technically this isn't a "runtime" attribute, but I think it's fine to bundle these together in this case

Reply:
I thought that "PipelinesApiRuntimeAttributes" in this case means "attributes which define the runtime environment (i.e., VM) which will be used by the certain PAPI Pipeline".